grpc/util.go

```diff
@@ -41,10 +41,15 @@ func ClientSetup(c *cmd.GRPCClientConfig) (*grpc.ClientConn, error) {
 	if err != nil {
 		return nil, err
 	}
+	if stats == nil {
```

Comment:
Looks like you removed the nil check, which will make this panic on the next line if stats is nil. This is probably a good place to explicitly check for nil and return error.